geomesa-hbase/geomesa-hbase-tools/bin/bootstrap-geomesa-hbase-aws.sh

```diff
@@ -58,8 +58,8 @@ echo "# Auto-registration for geomesa coprocessors ${NL}export CUSTOM_JAVA_OPTS=
 
 # Deploy the GeoMesa HBase distributed runtime to the HBase root directory
 if [[ "$ROOTDIR" = s3* ]]; then
-  aws s3 cp /opt/geomesa/dist/hbase/$DISTRIBUTED_JAR_NAME $ROOTDIR/lib/ && \
-  echo "Installed GeoMesa distributed runtime to $ROOTDIR/lib/"
+  aws s3 cp /opt/geomesa/dist/hbase/$DISTRIBUTED_JAR_NAME ${ROOTDIR%/}lib/ && \
+  echo "Installed GeoMesa distributed runtime to ${ROOTDIR%/}lib/"
```

Comment:
the `%/` removes the trailing slash, so you need to add it back in to the path. For consistency, I'd revert these two lines and just add a `ROOTDIR="${ROOTDIR%/}" # standardize to remove trailing slash` above this section, so that if there are any other file systems they benefit.

Reply:
Oh, I misunderstood what that did.